fairing/functions/function_shim.py

```diff
@@ -29,6 +30,14 @@ def get_execution_obj_type(obj):
 
     return ObjectType.NOT_SUPPORTED
 
+# compare the Python major and minor version for local and remote python
+def compare_version(local_python_version):
+    remote_python_version = ".".join([str(x) for x in sys.version_info[0:2]])
+    if local_python_version != remote_python_version:
+        logging.error("ERROR: The Python version mismatches with the local environment.")
+        sys.exit(1)
```

Comment:
throwing an exception makes sense here. sys.exit usually done in the entrypoint of the program like the main() not usually in a function.

Reply:
Addressed your comments. Thanks.